app/upgrades/v16/incentives.go

```diff
@@ -0,0 +1,34 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package v16
```

Comment:
super small nit, but this raises a warning in Goland and I love to not have any haha
```suggestion
// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)

package v16
```